components/cart/modal.tsx

```diff
@@ -67,7 +66,7 @@ export default function CartModal({ cart, cartIdUpdated }: { cart: Cart; cartIdU
             leaveFrom="opacity-100 backdrop-blur-[.5px]"
             leaveTo="opacity-0 backdrop-blur-none"
           >
-            <div className="fixed inset-0 bg-black/30" aria-hidden="true" />
```

Comment:
This removed the background fade. We'll want that back.

![CleanShot 2023-07-07 at 15 19 19@2x](https://github.com/vercel/commerce/assets/446260/282e9d00-d803-409f-8fdf-1aa69446c62b)

Reply:
I realize the Figma mock doesn't have this, but it add some separation.

Additionally, Figma has a border. Let's add one. 

![CleanShot 2023-07-07 at 15 30 00@2x](https://github.com/vercel/commerce/assets/446260/8bf951b8-0eda-4e75-8bab-c9b30f350f0a)